documentation.md

```diff
@@ -0,0 +1,25 @@
+# Documentation
```

Comment:
The document needs to actually state what types of information should be in github.com and why, as well as stating what types of information shouldn't.